frontend/src/components/Post/Post.tsx

```diff
@@ -0,0 +1,137 @@
+import React, {useState} from "react";
```

Comment:
```suggestion
import {useState} from "react";
```
remove unused import